tensorflow/python/keras/layers/convolutional.py

```diff
@@ -323,6 +323,13 @@ class Conv1D(Conv):
     kernel_constraint: Constraint function applied to the kernel matrix.
     bias_constraint: Constraint function applied to the bias vector.
 
+  Examples:
+    ```python
+    # Small convolutional model for 128-length vectors
+    model = Sequential()
```

Comment:
I would wish to be consistent with the example like this:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/layers/core.py#L553-L563
specifically, let user know what input shape and output shape is expected from the layer.

Reply:
Thank you. Adjusted the code accordingly.

Should there be an example for "variable timesteps", where input is `(None, 128)`?
Would be easy to add and probably help many users.